File: python/aula2/principal.py
from python.aula2.minhaBiblioteca import popular_aleatorio, popular_de_arquivo, exibir, copiarListaSemDuplicatas

lista_numerosA = []
lista_numerosB = []

popular_aleatorio(lista_numerosA, 100)
exibir(lista_numerosA)

#nome_arquivo = 'numeros.txt'
#popular_de_arquivo(lista_numerosB, nome_arquivo)
#exibir(lista_numerosB)

lista_resultado = []
copiarListaSemDuplicatas(lista_numerosB, lista_resultado)
exibir(lista_resultado)

File: cpp/aula2/principal.cpp
#include <iostream>
#include <ctime>
#include <cstdlib>
#include <string>
#include "minhaBiblioteca.h"

using namespace std;

int main() {

    //inicia o gerador de numeros aleatorios.
    srand(time(0));

    const int TAM = 100; // define tamanho fixo das listas

    int listaNumerosA[TAM];
    int listaNumerosB[TAM];
    int listaResultado[TAM];

    string nomeArquivo = "numeros.txt";
    popularAleatorio(listaNumerosA, TAM);
    exibir(listaNumerosA, TAM);

    popularArquivo(listaNumerosB, TAM, nomeArquivo);
    exibir(listaNumerosB, TAM);

    int quantidadeResultado = copiarListaSemDuplicatas(listaNumerosB, TAM, listaResultado, TAM);
    exibir(listaResultado, quantidadeResultado);

    return 0;
}